elasticsearch_exporter.go

```diff
@@ -267,52 +26,52 @@ type VecInfo struct {
 
 var (
 	gaugeMetrics = map[string]string{
-		"indices_fielddata_evictions":               "Evictions from field data",
-		"indices_fielddata_memory_size_in_bytes":    "Field data cache memory usage in bytes",
-		"indices_filter_cache_evictions":            "Evictions from field data",
-		"indices_filter_cache_memory_size_in_bytes": "Field data cache memory usage in bytes",
-		"indices_docs_count":                        "Count of documents on this node",
-		"indices_docs_deleted":                      "Count of deleted documents on this node",
-		"indices_store_size_in_bytes":               "Size of stored index data in bytes",
-		"indices_segments_memory_in_bytes":          "Memory size of segments in bytes",
-		"jvm_mem_heap_committed_in_bytes":           "JVM heap memory committed",
-		"jvm_mem_heap_used_in_bytes":                "JVM heap memory used",
-		"jvm_mem_heap_max_in_bytes":                 "JVM heap memory max",
-		"jvm_mem_non_heap_committed_in_bytes":       "JVM non-heap memory committed",
-		"jvm_mem_non_heap_used_in_bytes":            "JVM non-heap memory used",
+		"indices_fielddata_evictions":            "Evictions from field data",
+		"indices_fielddata_memory_size_bytes":    "Field data cache memory usage in bytes",
+		"indices_filter_cache_evictions":         "Evictions from field data",
+		"indices_filter_cache_memory_size_bytes": "Field data cache memory usage in bytes",
+		"indices_docs":                           "Count of documents on this node",
+		"indices_docs_deleted":                   "Count of deleted documents on this node",
+		"indices_store_size_bytes":               "Current size of stored index data in bytes",
+		"indices_segments_memory_bytes":          "Current memory size of segments in bytes",
+		"jvm_mem_heap_committed_bytes":           "JVM heap memory currently committed",
+		"jvm_mem_heap_used_bytes":                "JVM heap memory currently used",
+		"jvm_mem_heap_max_bytes":                 "JVM heap memory max",
+		"jvm_mem_non_heap_committed_bytes":       "JVM non-heap memory currently committed",
+		"jvm_mem_non_heap_used_bytes":            "JVM non-heap memory currently used",
 	}
 	counterMetrics = map[string]string{
 		"indices_flush_total":                   "Total flushes",
-		"indices_flush_time_in_millis":          "Cumulative flush time",
-		"transport_rx_count":                    "Count of packets received",
-		"transport_rx_size_in_bytes":            "Bytes received",
-		"transport_tx_count":                    "Count of packets sent",
-		"transport_tx_size_in_bytes":            "Bytes sent",
-		"indices_store_throttle_time_in_millis": "Throttle time for index store",
+		"indices_flush_time_ms_total":           "Cumulative flush time in milliseconds",
```

Comment:
What's inconsistent about that? Re/ seconds, I agree but didn't want to change this since I'm not sure we all can agree on that. I like having every time in seconds. @ewr: What do you think?